Tests/ProtocolBufferConversionTests.swift

```diff
@@ -462,6 +462,9 @@ final class ProtocolBufferConversionTests: XCTestCase {
     let memoData = Data([1, 2, 3])
     let memoFormat = Data([4, 5, 6])
     let memoType = Data([7, 8, 9])
+    let validated = true
+    let ledgerIndex = UInt32(1_000)
```

Comment:
can these be consts? they don't seem to be mutated at all

Reply:
let is the Swift version of const :)